Tests/Command/ExplainAdminCommandTest.php

```diff
@@ -121,10 +121,18 @@ protected function setUp()
                 return $adminParent;
             }));
 
-        $this->validatorFactory = $this->getMock('Symfony\Component\Validator\MetadataFactoryInterface');
+        // Symfony 2.x BC
```

Comment:
Expanded the comment a bit.  Basically the preference with this and the `interface_exists()` check in `CRUDControllerTest` is to use the Symfony 2.x "legacy" interfaces if they exist.